Makefile

```diff
@@ -21,6 +21,12 @@
 
 # VERSION is the source revision that executables and images are built from.
 VERSION ?= $(shell git describe --tags --always --dirty || echo "unknown")
+# GOLANG_IMAGE is the building golang container image used.
+GOLANG_IMAGE ?= public.ecr.aws/eks-distro-build-tooling/golang:1.19.6-3-gcc-al2
+# BASE_IMAGE_PRIMARY is the base layer image for the primary AWS VPC CNI plugin container
+BASE_IMAGE_PRIMARY ?= public.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base-iptables:latest.2
+# BASE_IMAGE_SECONDARY is the base layer image for the AWS VPC CNI init container and metrics publisher sidecar container
+BASE_IMAGE_SECONDARY ?= public.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base-glibc:latest.2
```

Comment:
I'm open to this alternative approach. I just want to call out we have three images instead of two. Other than the core container and init container, we also have a sidecar container for metrics. This means we will have three different BASE_IMAGE and three DOCKER_BUILD_FLAGS defined. 

My view on CNI init and CNI metrics is that they are both helpers that don't have much complex logic defined so will always share the same base image, hence it's ok to categorize both in the secondary bucket. Additionally, we can re-use the primary v.s. secondary flags in the future if more containers are introduced.

Reply:
Per discussion, updated the naming conventions.